### PR TITLE
MGMT-5419 Move disks creation code to assisted-service repo

### PR DESCRIPTION
**Pull request by osherdp**
Having less code on ``release`` repo is always good.
/cc @YuviGold @flaper87 
/hold

### CODE REVIEW

**Comment by osherdp** [comment]
/retest
(``packet-setup`` step failure)

File: deploy/operator/libvirt_disks.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+fi
+
+function create() {
+    export SIZE=${SIZE:-50G}
+
+    echo "Creating libvirt disks and attaching them..."
+    for node in ${NODES}; do
+        for disk in ${DISKS}; do
+            qemu-img create -f raw "/tmp/${node}-${disk}.img" "${SIZE}"
+            virsh attach-disk "${node}" "/tmp/${node}-${disk}.img" "${disk}"
```
Comment (YuviGold) [inline deploy/operator/libvirt_disks.sh:18]: Verify that the file doesn't exist
```suggestion
            img_path="/tmp/${node}-${disk}.img"
            if [ ! -f ${img_path} ]; then
                qemu-img create -f raw ${img_path} ${SIZE}
                virsh attach-disk "${node}" ${img_path} "${disk}"
            fi
```

File: deploy/operator/libvirt_disks.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        done
+    done
+
+    echo "Done destroying libvirt disks!"
+}
+
+case "$@" in
+    create)  create;;
+    destroy) destroy;;
+    *)       echo "Usage: DISKS=<disk names> bash ./libvirt_disks.sh (create|destroy)" && exit 1;;
```
Comment (YuviGold) [inline deploy/operator/libvirt_disks.sh:44]:
```suggestion
    *)       echo "Usage: DISKS=<disk names> bash ./libvirt_disks.sh ($(compgen -A function | paste -s -d'|')" && exit 1;;
```
will result all the available functions
```bash
Usage: DISKS=<disk names> bash ./libvirt_disks.sh (create|destroy)
```

File: deploy/operator/setup_lso.sh [hunk 3]
```diff
@@ -2,6 +2,12 @@ source utils.sh
 
 set -xeo pipefail
 
+if [ -z "${DISKS}" ]; then
+    echo "You must provide DISKS env-var. For example:"
+    echo "    bash DISKS=$(echo sd{b..f}) ./create_libvirt_disks.sh"
```
Comment (YuviGold) [inline deploy/operator/setup_lso.sh:7]:
```suggestion
    echo "    bash DISKS=\$(echo sd{b..f}) ./create_libvirt_disks.sh"
```

**Review by YuviGold — APPROVED** [review]
LGTM

**Review by flaper87 — COMMENTED** [review]
To be honest, I am not 100 % convince the `assisted-service` repo is the right place for this code. I would love to see us rely more on other existing tools like `dev-scripts`.

**Comment by osherdp** [comment]
@flaper87 I think this piece of code doesn't have any natural place to be
This is because we use both dev-scripts and test-infra in different cases, and because it's pretty much infrastructure-agnostic
So only reason to put here is for practical reasons, to enable ``dev-scripts``, ``release``, and ``assisted-test-infra`` to all have access to those scripts

**Pull request merged**